<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extra</class>
 <widget class="QWidget" name="extra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Colores</string>
   </property>
   <widget class="QComboBox" name="combo1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Centro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Punto</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lineas</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver lineas</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Punto</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tama√±o</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinner_tamano">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
